### DISCUSSION TITLE
Debug Mode Tips

### DISCUSSION THREAD

**Discussion by rmize** [Alkaar/resy-booking-bot#96] [General]
Anyone have tips for how to actually interpret the URL response that is returned in debug mode? For example I'm trying to see what reservation times / sizes / seating arrangements Don Angie typically releases. Thanks for the help

**Comment by Alkaar** [comment]
The return response is returned in what is called JSON format. Find a JSON formatter online and paste the response into that. It'll format it in a way that's a bit more readable. There's a hierarchy to it so hopefully, it'll be easier to figure it out.
